### PR TITLE
Added update inventory item endpoint

### PR DESCRIPTION
**Pull request by akash-belide**
Made the following changes :
- Added update inventory item endpoint in 'routes.py'
- Added test case for update endpoint

### CODE REVIEW

**Comment by vincezh2000** [comment]
Due to new merge to the master, you have unresolved conflcts. Please resolve them so that we can merge you PR to master.

File: service/routes.py [hunk 1]
```diff
@@ -109,6 +109,7 @@ def create_inventory_item():
     )
 
 
+
```
Comment (Ravi-Chavda) [inline service/routes.py]: Extra space

File: tests/test_models.py [hunk 2]
```diff
@@ -100,6 +130,7 @@ def test_read_a_item(self):
     #     items = InventoryItem.all()
     #     self.assertEqual(len(items), 5)
 
+
```
Comment (Ravi-Chavda) [inline tests/test_models.py]: Space

File: tests/test_routes.py [hunk 3]
```diff
@@ -139,6 +182,7 @@ def test_get_item(self):
     #     logging.debug("Response data = %s", data)
     #     self.assertIn("was not found", data["message"])
 
+
```
Comment (Ravi-Chavda) [inline tests/test_routes.py]: space

File: service/routes.py (hunk 1 above)
Comment (akash-belide) [reply service/routes.py]: Done

File: tests/test_models.py (hunk 2 above)
Comment (akash-belide) [reply tests/test_models.py]: Done

File: tests/test_routes.py (hunk 3 above)
Comment (akash-belide) [reply tests/test_routes.py]: Done

**Pull request merged**